src/app/cube/browse/browse.component.ts

```diff
@@ -59,12 +59,14 @@ export class BrowseComponent implements AfterViewInit, OnDestroy {
   filtering = false;
   filters: FilterSection[] = [
     {
+      title: 'collection',
```

Comment:
Are `name` and `title` synonymous? Why are they set to the same value?

Reply:
Resolved. I added a comment to the type at the bottom of the file.